### DISCUSSION TITLE
How do you rotate an imagine around it's center point?

### DISCUSSION THREAD

**Discussion by 22morph** [hajimehoshi/ebiten#1749] [Q&A] [answered]
I'm sorry if this is not the right place to ask but I don't know if another place. I would like to know how to rotate an image around it's center please.
I mean like in this [example](http://output.jsbin.com/owuyiq/)

**Comment by hajimehoshi** [comment, marked as answer]
Asking your question here is good since this is a signal indicating a lack of documentation :-)

https://github.com/hajimehoshi/ebiten/blob/master/examples/rotate/main.go

I think the rotate example explains what you want.

**Comment by 22morph** [comment]
Thank you, it does!

**Comment by eboatwright** [comment]
Thank you! :smile:
